File: programming-py/1.py
def two_sum(nums, target):
    for i in range(len(nums)):
        for j in range(i+1, len(nums)):
            if nums[i] + nums[j] == target:
                return [i, j]


# test below
# print(two_sum([2, 7, 15, 11], 9))
print(two_sum([2, 7, 15, 11], 3))

File: programming-swift/1.swift
func twoSum(nums: [Int], target: Int)-> [Int]{
    for i: Int in 0..<nums.count{
        for j: Int in i+1..<nums.count{
            if ( nums[i] + nums[j] == target){
                return [i,j];
            }
        }
    }
    return [];
}

// test below
print(twoSum(nums: [2, 7, 15, 11], target: 9))